v-next/hardhat/src/internal/builtin-plugins/network-manager/edr/utils/convert-to-edr.ts

```diff
@@ -212,23 +215,35 @@ export function edrRpcDebugTraceToHardhat(
   });
 
   // REVM trace adds initial STOP that Hardhat doesn't expect
+  // ASK: is this still valid?
   if (structLogs.length > 0 && structLogs[0].op === "STOP") {
     structLogs.shift();
   }
 
-  let returnValue = rpcDebugTrace.output?.toString("hex") ?? "";
+  let returnValue = debugTraceResult.output?.toString("hex") ?? "";
   if (returnValue === "0x") {
     returnValue = "";
   }
 
   return {
-    failed: !rpcDebugTrace.pass,
-    gas: Number(rpcDebugTrace.gasUsed),
-    returnValue,
+    failed: !debugTraceResult.pass,
+    gas: Number(debugTraceResult.gasUsed),
+    returnValue, // ASK: do we expect an unprefixed hex string here?
```

Comment:
This isn't resolved

Reply:
Can you answer this one? Depending on the response, we may be able to simplify the logic.